client/src/store/slices/userSlice/userSlice.js

```diff
@@ -21,6 +24,15 @@ export const fetchUserInfo = createAsyncThunk('user/feacthUserInfo', async (_, {
   }
 });
 
+export const fetchUserOrders = createAsyncThunk('user/fetchUserOrders', async (_, { rejectWithValue }) => {
+  try {
+    const { data } = await axiosConfig.get('/orders').then((orders) => orders);
+    return data;
+  } catch (err) {
+    return rejectWithValue(err.response);
+  }
+});
+
```

Comment:
we have orderSlice already created, i guess makes more sense to move the getOrder API to the orderSlice,

Reply:
it seems to me that this can be in the sense of both userSlice and orderSlice

Reply:
not a valid logic.. please refer back to toolkit documentation.